evaluation/README.md

```diff
@@ -1 +1,21 @@
-# Evaluate your app
+# Evaluate your AI app
+
+## Steps to run evaluation
```

Comment:
Better to be consistent to the datasets readme.

Reply:
Updated.